src/com/palmergames/bukkit/config/migration/ConfigMigrator.java

```diff
@@ -193,31 +189,39 @@ private void addPermissions(String key, String value) {
 	}
 
 	private List<Migration> readMigrator() {
-		try (Reader reader = new InputStreamReader(plugin.getResource(migrationFilename))) {
+		try (Reader reader = new InputStreamReader(JavaUtil.readResource(migrationFilename))) {
```

Comment:
Would this change not break other plugins' using the ConfigMigrator?

Reply:
It would, changed it to just check if the stream is null and log a message if so.